README.md

```diff
@@ -1,24 +1,34 @@
-# LaunchDarkly SDK Microservice
+> This project is not officially supported by LaunchDarkly.
 
-[![CircleCI](https://circleci.com/gh/launchdarkly/sdk-agent.svg?style=svg)](https://circleci.com/gh/launchdarkly/sdk-agent)
```

Comment:
Removing this because the link is appearing as broken, and the project is no longer available in CircleCI.